### PR TITLE
qcontainer: update failed_pattern of workaround machine type

### PR DESCRIPTION
**Pull request by PaulYuuu**
Update failed_pattern since error message changed.

ID: 1917404
Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

**Comment by PaulYuuu** [comment]
Latest error message on arm:
```
# /usr/libexec/qemu-kvm 
qemu-kvm: kvm_init_vcpu: kvm_arch_init_vcpu failed (0): Invalid argument
```

**Comment by PaulYuuu** [comment]
Hello @luckyh @zhenyzha , please help review, thanks.

**Review by zhenyzha — APPROVED** [review]

**Comment by zhenyzha** [comment]
(1/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.arm64-pci: PASS (912.13 s)
 (2/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.rh_kernel_update.arm64-pci: PASS (151.72 s)
 (3/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.boot.arm64-pci: PASS (47.70 s)

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

File: virttest/qemu_devices/qcontainer.py [hunk 1]
```diff
@@ -145,7 +145,8 @@ def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
                              shell=True,
                              verbose=False)
         # Some architectures (arm) require machine type to be always set
-        failed_pattern = r'(?:kvm_init_vcpu failed)|(?:machine specified)'
+        failed_pattern = (r'(?:kvm_init_vcpu.*failed)|(?:machine specified)|'
+                          r'(?:Invalid argument)')
```
Comment (luckyh) [inline virttest/qemu_devices/qcontainer.py:149]: Suggest `r'(?:kvm_init_vcpu.*Invalid argument)'`

**Comment by PaulYuuu** [comment]
sanity result:
```
import re
failed_pattern = r'(?:kvm_init_vcpu.*failed)|(?:machine specified)'
re.search(failed_pattern, 'qemu-kvm: kvm_init_vcpu: kvm_arch_init_vcpu failed (0): Invalid argument')
<_sre.SRE_Match object; span=(10, 50), match='kvm_init_vcpu: kvm_arch_init_vcpu failed'>
```

**Review by luckyh — APPROVED** [review]
LGTM

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**